rest/model/test.go

```diff
@@ -43,7 +43,9 @@ type TestLogs struct {
 	URLLobster *string `json:"url_lobster,omitempty"`
```

Comment:
Could you also update the unit tests in `rest/model/test_test.go` to assert that these new fields are set properly?